README.md

```diff
@@ -95,6 +95,60 @@ Other files you'll usually want to select `n` on - particularly those files plug
 
 However, files that are _not_ commonly edited by plugin authors may deserve a diff check (`d`) if you've made changes to these sorts of files. For example, anything in `scripts/`.
 
+### Migrating to the Latest Version
+
+> **NOTE:** This section comes with the caveat that you know your project better than we do; so, do what you think is best.
+
+Migrating across major versions can be a bit of a pain sometimes, but we're working on making it better with the generator. Here are some notes on migrating to the current major version.
+
+You may - depending on your customizations - want to prepare your project by doing one of two things.
+
+If you have made no customizations, the simplest process may be to remove some or all of the following:
+
+- `dependencies`, `devDependencies`, and/or `scripts` from `package.json`
+- `package-lock.json`
```

Comment:
Scripts in the `scripts/*` directory as well as `test/karma.conf.js`.  That may apply to the method below this as well

Reply:
Also, `test/index.html`. Added it.